README.md

```diff
@@ -116,6 +116,29 @@ But it will not be performed in these scenarios
 * `where{ !(id == '123') }`
 * <code>where{ (id == '123') &#124; (bar == 'baz') }</code>
 
+## Combination of Filters
+Multiple DAW filters can be combined. This will provides the ability to compose
+filtering conditions to keep your code more readable and DRY.
+
+### Combining conditions with AND
+```ruby
+class Foo
+  include Dynamoid::Document
+
+  field :bar
+  field :baz
+end
+
+filter1 = Foo.where{ bar == 'abcd' }
+filter2 = Foo.where{ baz == 'dude' }
+
+# All of these produce the same results
+combination1 = filter1.where(filter2)
+combination2 = filter1.and(filter2)
+combination3 = filter1 + filter2
```

Comment:
I'm not sure about the `+` I see that as an `.or`

Reply:
I agree with @nrowegt when I think of it semantically; + is a combination, * would be a intersection

Reply:
Great point, I'll remove it